concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1569,6 +1569,7 @@ jobs:
               - |
                 echo "RUNNING THIS ACTION FOR AZ-A, WILL MEAN THIS CONCOURSE IS BEING USELESS!"
                 echo "You will need to run the 'revert' action by hand, to get it back to working state."
+                # ^ is this still true?
```

Comment:
Shouldn't be true any more, I don't think.

Reply:
Cool I'll remove